azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1585,7 +1587,12 @@ private void handleProjectPage(final HttpServletRequest req,
           page.add("exec", false);
         }
 
-        final List<Flow> flows = project.getFlows();
+        final List<Flow> flows = new ArrayList<>();
+        for (final Flow flow : project.getFlows()) {
```

Comment:
nit: This can be made short by `project.getFlows().forEach()`

Reply:
I'll improve it, thanks!